File: app/src/main/java/com/example/myapplication/activity/FullImage.java
package com.example.myapplication.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.example.myapplication.R;

public class FullImage extends AppCompatActivity {

    private ImageView imageView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_full_image);

        imageView = findViewById(R.id.fullImage);
        Glide.with(this).load(getIntent().getStringExtra("url"))
                .into(imageView);
    }
}

File: app/src/main/java/com/example/myapplication/activity/FullImage.kt
package com.example.myapplication.activity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ImageView
import com.example.myapplication.R
import com.bumptech.glide.Glide

class FullImage : AppCompatActivity() {
    private var imageView: ImageView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_full_image)
        imageView = findViewById(R.id.fullImage)
        val url = intent.getStringExtra("url")
        Glide.with(this).load(url).into(imageView!!)
    }
}